core/trino-spi/src/main/java/io/trino/spi/ptf/TableFunctionSplitProcessor.java

```diff
@@ -24,17 +22,14 @@
  * Thread-safety: implementations do not have to be thread-safe. The {@link #process} method may be called from
  * multiple threads, but will never be called from two threads at the same time.
  */
+@Experimental(eta = "2023-03-31")
 public interface TableFunctionSplitProcessor
 {
     /**
      * This method processes a split. It is called multiple times until the whole output for the split is produced.
      *
-     * @param split a {@link ConnectorSplit} representing a subtask, or {@code null} if a split has already started to be processed,
-     * and the implementation returned a {@link TableFunctionProcessorState.Processed} with
-     * {@link TableFunctionProcessorState.Processed#isUsedInput()} being {@code true}.
      * @return {@link TableFunctionProcessorState} including the processor's state and optionally a portion of result.
-     * After the returned state is {@code FINISHED}, the method will not be called again for the split currently being processed
-     * and may be called with a new split.
```

Comment:
This was actually incorrect, broken in https://github.com/trinodb/trino/pull/16955